hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfs/hdfs.c

```diff
@@ -2896,7 +2903,7 @@ hdfsGetHosts(hdfsFS fs, const char *path, tOffset start, tOffset length)
     for (i = 0; i < jNumFileBlocks; ++i) {
         jFileBlock =
             (*env)->GetObjectArrayElement(env, jBlockLocations, i);
-        if (!jFileBlock) {
+        if ((*env)->ExceptionOccurred || !jFileBlock) {
```

Comment:
this should be ExceptionOccurred(env), right? it's a function, not a variable, isn't it?

(same below a few places)

Reply:
Yeah, it definitely should be. Fixed.